phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="Project Rules">
+    <rule ref="Newfold"/>
+    <config name="testVersion" value="7.0-"/>
+    <config name="minimum_supported_wp_version" value="5.8"/>
```

Comment:
Our minimum supported WP version is 6.2 - https://github.com/newfold-labs/wp-plugin-hostgator/blob/main/.github/workflows/cypress-matrix.yml#L31